getgather/mcp/dpage.py

```diff
@@ -30,28 +31,44 @@
 )
 from getgather.logs import logger
 from getgather.mcp.html_renderer import DEFAULT_TITLE, render_form
+from getgather.zen_distill import (
+    autoclick as zen_autoclick,
+    capture_page_artifacts as zen_capture_page_artifacts,
+    distill as zen_distill,
+    get_new_page,
+    init_zendriver_browser,
+    page_query_selector,
+    run_distillation_loop as zen_run_distillation_loop,
+    terminate_zendriver_browser,
+    zen_report_distill_error,
+)
 
 router = APIRouter(prefix="/dpage", tags=["dpage"])
 
 
-active_pages: dict[str, Page] = {}
+active_pages: dict[str, Page | zd.Tab] = {}
```

Comment:
A union like this, so we don't need to keep track of Patchright pages and Zendriver tabs separately, and thus we can use the same `/dpage` route (for the sign-in link).